sentence_transformers/evaluation/NanoBEIREvaluator.py

```diff
@@ -420,6 +420,8 @@ def _load_dataset(self, dataset_name: DatasetNameType, **ir_evaluator_kwargs) ->
         )
 
     def _validate_dataset_names(self):
+        if self.dataset_names == []:
```

Comment:
```suggestion
        if len(self.dataset_names) == 0:
```
This is a personal preference, but I prefer checking lengths like this :)

Reply:
done!

Reply:
on second thought, theres also this case when datasets = [''], an empty string, 

currently it recognizes as an invalid dataset
```
ValueError: Dataset(s) [''] not found in the NanoBEIR collection.Valid dataset names are: ['climatefever', 'dbpedia', 'fever', 'fiqa2018', 'hotpotqa', 'msmarco', 'nfcorpus', 'nq', 'quoraretrieval', 'scidocs', 'arguana', 'scifact', 'touche2020']
```

would it be nicer to consider it as an empty dataset and use this check instead?

Reply:
I don't think we have to worry about this case, I don't think many people would use `[""]` and if they do, the error is already pretty descriptive.

Reply:
cool then